flu_ar2/main.py

```diff
@@ -16,7 +16,10 @@
 )
 def main(today_date: str | None = None):
     """Get clade counts and save to S3 bucket."""
-    today_date = datetime.date.fromisoformat(today_date)
+    try:
+        today_date = datetime.date.fromisoformat(today_date)
+    except (TypeError, ValueError):  # if today_date is None or a bad format
+        today_date = datetime.date.today()
```

Comment:
another option, which i am taking in the next model, is just to require that the `today_date` argument be passed?